tests/rbd/rbd_utils.py

```diff
@@ -722,6 +727,20 @@ def set_config(self, level, entity, key, value, **kw):
         """
         return self.exec_cmd(cmd=f"rbd config {level} set {entity} {key} {value}", **kw)
 
+    def trash_purge(self, pool_name, **kw):
+        """
+        Purge/Remove all expired images in trash from the pool
+        Args:
+            pool_name: name of the pool
+            kw: any other optional arguments
+         Returns:
+        """
+        cmd = f"rbd trash purge --pool {pool_name}"
+        if kw:
```

Comment:
`Kw` contains lot of things, what is the purpose of checking that here,
Can you please add doc string what exactly we are looking in `if` condition?

Reply:
kw when passed to this method in my case does not contain a lot of things. I am expecting it to contain only any additional arguments required for rbd trash purge command. So I am checking if anything is present and kw is not None. 

That's why the check if kw.